teamsupport/__init__.py

```diff
@@ -1,11 +1,17 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
+import teamsupport.config as config
 from teamsupport.models import Action, Ticket, User
 from teamsupport.services import TeamSupportService
 
 __author__ = 'Yola Engineers'
 __email__ = 'engineers@yola.com'
-__version__ = '0.1.3'
+__version__ = '0.1.4'
 
-__all__ = (Action, TeamSupportService, Ticket, User)
+__all__ = (Action, TeamSupportService, Ticket, User, config)
+
+
+def init(org_id, auth_key):
```

Comment:
`init` makes it sound like it's doing work. `configure`? `login`? (ok, that option also sounds like it's doing work...)

These could also be parameters to `TeamSupportService`, but then that would have to be passed to the other classes constructors.